### PR TITLE
.NET 6 Support

### PR DESCRIPTION
**Pull request by JosunLP**
A complete migration to .NET 6 LTS

### CODE REVIEW

**Comment by igorkis-scrts** [comment]
Thanks for PR! I will take a look at it today.

File: A3ServerTool/A3ServerTool.csproj [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-      <Version>1.1.19</Version>
+      <Version>1.1.39</Version>
     </PackageReference>
     <PackageReference Include="MvvmLightStd10">
       <Version>5.4.1.2</Version>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
       <Version>13.0.1</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.4.336902">
```
Comment (igorkis-scrts) [inline A3ServerTool/A3ServerTool.csproj:27]: Could this be deleted as upgrading process already happened in your PR?

**Review by igorkis-scrts — COMMENTED** [review]
I will do some tests this week to see if nothing breaks after .NET 6 transition and if everything is okay then PR will be merged and new version will be released.
Thanks for contribution!

File: A3ServerTool/A3ServerTool.csproj (hunk 1 above)
Comment (JosunLP) [reply A3ServerTool/A3ServerTool.csproj:27]: yep

**Pull request merged**